### PR TITLE
docs: Fix formatting/style on InSpec DSL page

### PR DESCRIPTION
**Pull request by jerryaldrichiii**
This fixes several formatting/style issues on https://www.inspec.io/docs/reference/dsl_inspec/

### CODE REVIEW

File: docs/dsl_inspec.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ where
 * `ref` is a reference to an external document
 * `describe` is a block that contains at least one test. A `control` block must contain at least one `describe` block, but may contain as many as required
 * `sshd_config` is an InSpec resource. For the full list of InSpec resources, see InSpec resource documentation
-* `its('Port')` is the matcher; `{ should eq('22') }` is the test. A `describe` block must contain at least one matcher, but may contain as many as required
+* `its('Port')` is the matcher; `{ should eq '22' }` is the test. A `describe` block must contain at least one matcher, but may contain as many as required
```
Comment (clintoncwolfe) [inline docs/dsl_inspec.md:55]: I mean, this is literally wrong, but now it's well-formatted and wrong, so ... :100:

**Review by clintoncwolfe — APPROVED** [review]
I have a markdown parser I think you'd like to meet.

File: docs/dsl_inspec.md (hunk 1 above)
Comment (jerryaldrichiii) [reply docs/dsl_inspec.md:55]: Hah! Yeah didn't even read for content. Quite the hilarity though second time through.

Comment (jerryaldrichiii) [reply docs/dsl_inspec.md:55]: /me gestures to the can of worms we are avoiding :smile:

**Review by miah — APPROVED** [review]
Thanks @jerryaldrichiii

**Pull request merged**